src/centos/stream8/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN dnf install --setopt tsflags=nodocs --refresh -y \
         hostname \
         jq \
         krb5-devel \
+        libatomic \
```

Comment:
Can we add a comment about the dependency?

Reply:
Maybe! Let's see